inference-engine/tests/functional/plugin/cpu/single_layer_tests/reduce_ops.cpp

```diff
@@ -155,186 +241,421 @@ const std::vector<ngraph::helpers::ReductionType> reductionLogicalTypes = {
         ngraph::helpers::ReductionType::LogicalAnd
 };
 
-const std::vector<std::vector<size_t>> inputShapes = {
-        std::vector<size_t>{10, 5, 15, 12},
-        std::vector<size_t>{3, 5, 7, 9},
+std::vector<std::vector<ov::test::InputShape>> inputShapes = {
+    {{{}, {{2, 19, 2, 9}}}},
+    {{{-1, {2, -1}, -1, -1}, {{2, 9, 2, 9}, {2, 19, 2, 9}, {2, 9, 2, 15}}}}
```

Comment:
Fixed. Thanks Maksim!

Reply:
I merged upper bound tests cases with  those of lower bound, to avoid timeout  caused by too much test cases, in  ci test machines with more strict total test time limitation. Further discussion is always welcomed, if you have any concern. Thanks Maksim!